validation/markets-validator.js

```diff
@@ -102,17 +107,70 @@ fs.readdir(MARKETS_DIR_PATH, function (err, files) {
 });
 
 process.on('beforeExit', function () {
+    console.log("\n");
     asyncWarnings.forEach(function (warning) {
-        console.log('Warning: %s', warning.toString());
+        console.log("Warning: ".warning + getFormattedText(warning.toString()));
```

Comment:
This function swallows information in some cases:

```js
getFormattedText(`\
usingen: HTTP response status of data source url was: 302
     --> New location: https://www.usingen.de/pdfs/satzungen/marktordnung-der-stadt-usingen.pdf?cid=3c`)
```

Reply:
I don't understand. When I output the warnings without piping them through `getFormattedText()` I see the same text, e.g.:

``` bash
Warning: basel: HTTP response status of data source url was: 303
     --> New location: /error_path/400.html?al_req_id=YGwlcw9zrmd@nDkUqhTmLgAAA5A
Warning: kassel: HTTP response status of data source url was: 403
Warning: hamburg: HTTP response status of data source url was: 500
Warning: solingen: HTTP response status of data source url was: 403
```

Reply:
@derhuerst Did you see my comment here?

Reply:
@derhuerst Fixed in f99eb2922a285dd178f1153fd566d7cab21cc92b. Thanks for spotting it.